rest/src/main/java/org/jboss/pnc/rest/provider/ProjectProvider.java

```diff
@@ -103,7 +103,8 @@ public Integer update(Integer id, ProjectRest projectRest) throws ConflictedEntr
 
     private void validateBeforeSaving(ProjectRest projectRest) throws ConflictedEntryException {
         Project project = projectRepository.queryByPredicates(withProjectName(projectRest.getName()));
-        if(project != null) {
+        //don't validate against myself
```

Comment:
@thescouser89 Probably because of this comment :D It was validating against itself but from DB.... argh... silly mistake..